### PR TITLE
Test PR to summarise customisations for Up2U

### PR DESCRIPTION
**Pull request by michzimny**

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -7,5 +7,12 @@ docker run \
 	-v ${PWD}/logo/logo.svg:${BASEDIR}/${MM}/app/resources/images/logo.svg \
 	-e BASEDIR=${BASEDIR} -e MM=${MM} \
 	--network host \
+	--restart unless-stopped \
+	--name mm \
 	--detach \
       misi/mm
+
```
[the remark this answers is not in the document]
Comment (michzimny) [reply run.sh]: No, probably not. I don't remember the reason... I will provide a single file, but then, perhaps, HTML/CSS must take care of proper sizes of images (larger in the front page, smaller in the room).

**Comment by michzimny** [comment]
I will try next week, but I'm strongly involved in the planning activity we do for EC with the deadline next week...

File: run.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (michzimny) [reply run.sh]: @misi, I just removed one of the logo files.

**Comment by michzimny** [comment]
@misi, could you propose how to customize your image? Perhaps the logo path could be provided via an environmental variable?

**Comment by misi** [comment]
@michzimny There is many ways we can do it. 
1. ansible way
    The way that I do with ansible script is the following
    1. git clone master branch
    2. replace image, customize config files
    3. run the image
2. branch
    But if you like we can create a branch where you can have all the customized theme and config and you only need to merge the upstream changes time to time..

**Comment by misi** [comment]
I checked it, and merged what I think needed, so I will close it now. 
If needed let me know and I reopen it.

**Pull request closed without merging**